js/script.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+$(function() {
+
+    function checkUser(e, elements, users) {
+        let active = elements.activeName;
+        if(e.keyCode === 40){
+            if(elements.isOpen && active >= 0 && active < 9){
+                elements.activeName = active + 1;
+                }else {
+                elements.activeName = 0;
+                }
+            return makeElemActive(active, elements.activeName)
+            }
+            if(e.keyCode === 38) {
+                if (elements.isOpen && active > 0 && active <= 9) {
+                    elements.activeName = active - 1;
+                    } else {
+                    elements.activeName = 9;
+                    }
+                return makeElemActive(active, elements.activeName);
+            }
+            if (e.keyCode === 13 && elements.isOpen && elements.activeName){
+            e.preventDefault();
+            let text = $('.selectUser:eq(' + elements.activeName + ')').text();
+                return fillName(null, elements, text);
+            }
+
+        setTimeout(function () {
+            elements.cursorPosition = elements.textArea.prop("selectionStart");
+            if(elements.cursorPosition){
+                let value = elements.textArea.val();
+                elements.startPos = getStartPosition(value, elements.cursorPosition-1);
+                elements.isOpen = false;
+                if(elements.startPos){
+                    let text = value.substring(elements.startPos, elements.cursorPosition);
+                      return  find(text, users, elements);
+                }
+                return elements.showUsers.empty();
+                }
+        }, 0);
+
+    }
+
+    function makeElemActive(oldIndex, newIndex) {
+        $('.selectUser:eq(' + oldIndex + ')').blur();
+        $('.selectUser:eq(' + newIndex + ')').focus();
+    }
+    function getStartPosition(str, pos) {
+        if (str.substring(pos, pos + 1) === '@') {
+            return pos + 1;
+        } else if (pos === 0) {
+            return 0
+        }
+            return getStartPosition(str, pos - 1);
+    }
+
+    function getEndPosition(str, pos) {
+        if (!str.substring(pos, pos + 1).match(/[A-Za-z]/)) {
```

Comment:
@lianatumikyan maybe here you can write inline 'if' like this?
return !str.substring(pos, pos + 1).match(/[A-Za-z]/) ? pos : getEndPosition(str, pos + 1);